selftests/unit/test_libvirt_xml.py

```diff
@@ -814,10 +815,10 @@ def test_arbitrart_attributes(self):
         channel = librarian.get('channel')(virsh_instance=self.dummy_virsh)
         console = librarian.get('console')(virsh_instance=self.dummy_virsh)
         for chardev in (parallel, serial, channel, console):
-            attribute1 = utils_misc.generate_random_string(10)
-            value1 = utils_misc.generate_random_string(10)
-            attribute2 = utils_misc.generate_random_string(10)
-            value2 = utils_misc.generate_random_string(10)
+            attribute1 = data_factory.generate_random_string(10)
```

Comment:
Please use a separate PR to hold the replacements of any other functionalities unrelated to disk provided by `utils_mise`, thanks.

Reply:
sure, I will do that